### PR TITLE
Adds new tips for Xenomorph.

### PR DESCRIPTION
**Pull request by HeresKozmos**
# About the pull request

Adds 17 new tips for Xenomorph.

# Explain why it's good for the game

Knowledge is good.

# Changelog

:cl:
add: Added over a dozen new tips for playing Xenomorph.
/:cl:

### CODE REVIEW

**Comment by HeresKozmos** [comment]
Changed some wording as suggested to me, also added a new tip.

**Comment by HeresKozmos** [comment]
More tips for more knowledge to cram into your craniums

**Comment by HeresKozmos** [comment]
i need a proof reader...

File: strings/xenotips.txt [hunk 1]
```diff
@@ -37,3 +37,21 @@ You can filter out the Xenomorphs displayed in hive status by health, allowing y
...
 As a Xenomorph, the list of available tunnels is sorted by their distance to the player!
 As a Xenomorph, pay attention to what a marine is wearing. If they have no helmet, aiming for the head will be significantly more damaging, if they aren't wearing gloves, then think about going for their hands.
+Applying acid to flares makes them burn faster.
+Melting weapons, ammo or other valuables owned by marines is a good way to debilitate them.
+The secrete acid shortcut is (by default) Shift + C.
+Placing acid traps under map debris such as metal sheets, wooden planks or other semi-large to large items is good for catching marines by surprise.
+Burrower can place tunnels underneath tables or lockers.
+Remember that Lessers and Huggers cannot speak or use the Hivemind for three minutes after spawning.
+Alt-clicking on the Queen tracker allows you to set it to also track the Hive Core, any living leaders or any existing tunnels and not just the Queen. Tunnels are ordered by closest to furthest; top to bottom.
```
Comment (Birdtalon) [inline strings/xenotips.txt:46]:
```suggestion
Alt-clicking on the Queen tracker allows you to set it to also track the Hive Core, any living leaders or any existing tunnels. Tunnels are ordered by closest to furthest; top to bottom.
```

File: strings/xenotips.txt [hunk 2]
```diff
@@ -37,3 +37,21 @@ You can filter out the Xenomorphs displayed in hive status by health, allowing y
...
+Be mindful of not body-blocking the escape of any fleeing sisters; this is especially true if they are heavily injured.
```
Comment (Birdtalon) [inline strings/xenotips.txt:47]: Don't particularly like how this is worded.

File: strings/xenotips.txt [hunk 3]
```diff
@@ -37,3 +37,21 @@ You can filter out the Xenomorphs displayed in hive status by health, allowing y
...
+As a Xenomorph, you are able to melt advanced body scanners, sleepers, MarineMeds and limb printers.
+You can break roller beds with a single slash to stop a medic from quickly transporting a body.
+As a Defender, lowering your crest gives you increased resistance to stuns from explosives and other stuns such as shotgun blasts.
```
Comment (Birdtalon) [inline strings/xenotips.txt:50]:
```suggestion
As a Defender, lowering your crest gives you increased resistance to stuns like explosives or shotgun shells.
```

File: strings/xenotips.txt [hunk 4]
```diff
@@ -37,3 +37,21 @@ You can filter out the Xenomorphs displayed in hive status by health, allowing y
...
+Tail stab will always hit the targetted limb and will do 1.2x your normal melee damage; it has a two tile range.
+If you suspect a door is shocked, you can tail stab it to check and receive a massively reduced stun than if you would have triggered it normally.
```
Comment (Birdtalon) [inline strings/xenotips.txt:52]: This could be be more concise.

File: strings/xenotips.txt [hunk 5]
```diff
@@ -37,3 +37,21 @@ You can filter out the Xenomorphs displayed in hive status by health, allowing y
...
+You can use your tail stab to slowly damage and break doors; this takes some time.
+Some smaller Xenomorphs such as Runners can hide behind barricades by resting against them, a good surprise tactic.
+Hivelords or Defenders with a lowered crest can open any existing holes in walls created by others such as Warriors; it may take several attempts to fully open the wall.
```
Comment (Birdtalon) [inline strings/xenotips.txt:55]: Probably better to either make the tip the generic "some xenomorphs" or make it specific to defender. Also no need to have "such as warriors".

File: strings/xenotips.txt (hunk 2 above)
Comment (HeresKozmos) [reply strings/xenotips.txt:47]: all four lines or one specific one?

File: strings/xenotips.txt (hunk 4 above)
Comment (HeresKozmos) [reply strings/xenotips.txt:52]: again, a specific one or all four?

File: strings/xenotips.txt (hunk 5 above)
Comment (HeresKozmos) [reply strings/xenotips.txt:55]: good point

File: strings/xenotips.txt [hunk 6]
```diff
@@ -37,3 +37,21 @@ You can filter out the Xenomorphs displayed in hive status by health, allowing y
 Each xeno has their own ‘tackle counter’ on a marine. The range to successfully tackle can be anywhere from two to six tackles based on caste. If a marine gets stunned or knocked over by other means it will reset everyone's tackle counters and they may get up!
 As a Xenomorph, the list of available tunnels is sorted by their distance to the player!
 As a Xenomorph, pay attention to what a marine is wearing. If they have no helmet, aiming for the head will be significantly more damaging, if they aren't wearing gloves, then think about going for their hands.
+Applying acid to flares makes them burn faster.
+Melting weapons, ammo or other valuables owned by marines is a good way to debilitate them.
+The secrete acid shortcut is (by default) Shift + C.
+Placing acid traps under map debris such as metal sheets, wooden planks or other semi-large to large items is good for catching marines by surprise.
+Burrower can place tunnels underneath tables or lockers.
+Remember that Lessers and Huggers cannot speak or use the Hivemind for three minutes after spawning.
```
Comment (Birdtalon) [inline strings/xenotips.txt:45]:
```suggestion
Lesser drones and facehuggers cannot speak or use the Hivemind for three minutes after spawning.
```

**Review by Birdtalon — APPROVED** [review]

**Pull request merged**